crates/rpc/rpc/src/debug.rs

```diff
@@ -288,11 +279,16 @@ where
 
                     let to_db_service = self.spawn_js_trace_service(at, maybe_override_db)?;
 
-                    let mut inspector = JsInspector::new(code, config, to_db_service)?;
-                    let (res, env) = inspect(db, env, &mut inspector)?;
-
-                    let result = inspector.json_result(res, &env)?;
-                    Ok(GethTrace::JS(result))
+                    Ok(GethTrace::JS(
+                        self.inner
```

Comment:
I'd like to do the return in two steps

let res = ...

Ok(GethTrace::JS(res))

just looks nicer

Reply:
@mattsse Sure, done